docker-compose.yaml

```diff
@@ -29,6 +29,7 @@ x-airflow-common: &airflow-common
     - ${AIRFLOW_PROJ_DIR}/plugins:/opt/airflow/plugins
     - ${AIRFLOW_PROJ_DIR}/config:/opt/airflow/config
     - ${AIRFLOW_PROJ_DIR}/airflow.cfg:/opt/airflow/airflow.cfg
+    - knack-certs:/opt/airflow/certs/knack
```

Comment:
none of the containers in the docker-compose stack does not need this volume, however i have learned that docker compose will not create the volume unless at least on services uses it (this is not documented 😕 ). in our case, our airflow scheduler is going to spin up other containers (the 311 ETL) which **do** need the volume.

the work around to this is to create an external volume outside of any docker compose file, but then that external volume becomes a dependency of our stack 🤷‍♂️

Reply:
Good solution here John. I agree with you that that is a pretty unfortunate hole in the documentation for `docker compose`. I think you're 100% picking the best thing to include this in the airflow stack because it has the job of providing whatever is needed for ETLs to do their business. 👍

Reply:
+1 - this makes sense to me too and adding a certs folder makes a lot of sense in case we see this need come up again. Seeing the mount param in the `DockerOperator` makes it clear to trace what is going to me too.

Reply:
thanks for the feedback, y'all!!!